test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/StartControllerSpec.scala

```diff
@@ -1489,7 +1490,7 @@ class StartControllerSpec
                 Right(
                   Some(
                     SessionData.empty.copy(
-                      journeyStatus = Some(sample[JourneyStatus])
+                      journeyStatus = Some(sample(implicitly[ClassTag[JourneyStatus]], journeyStatusGen))
```

Comment:
this is annoying but I couldn't figure out a fix. What's happening is that there's an implicit `Gen[JourneyStatus]` in scope but there are also `Gen` instances for the subtypes of `JourneyStatus` also in scope (e.g. `Gen[Subscribed]`, `Gen[SubscriptionReady]`) - the compiler is getting confused and saying that `Gen[JourneyStatus]` conflicts with those other `Gen` instances. Not sure what the best way to fix that was, but this is a workaround.